pkg/artifactory/validators.go

```diff
@@ -477,6 +479,16 @@ var projectKeyValidator = validation.ToDiagFunc(
 	validation.StringMatch(regexp.MustCompile(`^[a-z0-9]{3,10}$`), "project_key must be 3 - 10 lowercase alphanumeric characters"),
 )
 
+func repoLayoutRefSchemaOverrideValidator(_ interface{}, _ cty.Path) diag.Diagnostics {
```

Comment:
more appropriate might be `alwaysFailValidator` - but I think this PR has been tortured enough